golang/timers/README.md

```diff
@@ -0,0 +1,7 @@
+# mattermost-apps-golang-timers
+
+## Install & Usage
+
+Refer to the [Hello World in Go Quick Start Guide](https://developers.mattermost.com/integrate/apps/quickstart/quick-start-go/) for instructions on how to use this example.
```

Comment:
This guide explains how to use the `Hello World` app specifically so it's probably confusing to send the reader there because it will send them into another folder in the repo. Instead, either a standalone Go tutorial page for the timer should added.

Even if that document page doesn't get made yet, I still think you should just link them to the [Developer Setup Guide](https://developers.mattermost.com/integrate/apps/quickstart/) instead so you can be assured they have Mattermost running with your correct dev setup for the example.